assignments/assignment_4/README.md

```diff
@@ -7,16 +7,17 @@ The focus of this (final) assignment is about OOP and design patterns. And we ha
 ## 1- Design Patterns
 
 1. Copy the JSON data from the `assignment_4` folder to the `life_expectancy/data` folder.
-2. Refactor your code so that is can accept data in other formats if necessary. You can use the [Strategy pattern](https://refactoring.guru/design-patterns/strategy) for this, but you can also use a different one if you prefer.
-3. Ensure you still have complete test coverage and a high `pylint` score.
+2. Refactor your code so that is can accept and clean data in other formats if necessary. You can use the [Strategy pattern](https://refactoring.guru/design-patterns/strategy) for this, but you can also use a different one if you prefer.
```

Comment:
```suggestion
2. Refactor your code so that it can accept and clean data in other formats if necessary. You can use the [Strategy pattern](https://refactoring.guru/design-patterns/strategy) for this, but you can also use a different one if you prefer.
```